app/serializers/web_hooks_post_serializer.rb

```diff
@@ -0,0 +1,21 @@
+class WebHooksPostSerializer < PostSerializer
+  def include_yours?
```

Comment:
Use `include_` to exclude some UI specific fields or building from basic_serializer?